ci-operator/step-registry/cucushift/installer/rehearse/baremetalds/ipi/ovn/ipv4/kubevirt/cucushift-installer-rehearse-baremetalds-ipi-ovn-ipv4-kubevirt-workflow.yaml

```diff
@@ -0,0 +1,11 @@
+workflow:
+  as: cucushift-installer-rehearse-baremetalds-ipi-ovn-ipv4-kubevirt
```

Comment:
I saw `cucushift-installer-rehearse-baremetalds-ipi-ovn-ipv4-kubevirt-hypershift` set `DEVSCRIPTS_CONFIG` env, this workflow does not need those settings?

Reply:
yep, Sorry, i forget it, update done